TimerGUI.pyw

```diff
@@ -1,120 +1,97 @@
-#GUI and commands code
-
-import tkinter as tk
-import time
-import math
+#program config (in seconds):
+#time before first window pop's out 
+programInitialDelay = 480
+#time before first window pop's out again after clicking NO
+shortHideButtonDelay = 120
+# !!!CHANGE NOT ADVISED!!! time before "change log file" info pop's out again
+timerDelay = 3600
 
-#first window creator 
-def createInitialWindow():
-    #first window constructor
-    initialWindow = tk.Tk()
-    initialWindow.title('Timer lasera')
+#GUI and commands code
+import tkinter as tk, time, math
 
-    #Set window to appear in the middle
+def createWindow():
+    
+    window = tk.Tk()
+    window.title('Timer lasera')
+    
     windowWidth = 330
     windowHeight = 122
```

Comment:
Might be worth to pass width, height and position in a structure as parameter for a generic `createWindow` method